src/core/runner.ts

```diff
@@ -417,6 +420,11 @@ export default class Runner {
       journey.callback({ params: options.params } as any);
       journey.monitor.update(this.monitor?.config);
       journey.monitor.validate();
+      if (journey.skip && !options.force) {
+        throw new Error(
+          `Journey ${journey.name} is skipped. Remove the skip method or use -f, --force to ignore this.`
```

Comment:
It looks like you are trying to push skipped monitors (journey.name), Please remove the .skip annotation and try again.